tests/Integration/ResponseStatusCodeTest.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace DDTrace\Tests\Integration;
+
+use DDTrace\Tests\Common\SpanAssertion;
+use DDTrace\Tests\Common\WebFrameworkTestCase;
+use DDTrace\Tests\Frameworks\Util\Request\GetSpec;
+
+class ResponseStatusCodeTest extends WebFrameworkTestCase {
+    protected static function getAppIndexScript() {
+        return __DIR__ . '/ResponseStatusCodeTest_files/index.php';
+    }
+
+    protected static function getEnvs() {
+        return array_merge(parent::getEnvs(), [
+            'DD_TRACE_NO_AUTOLOADER' => '1',
+            'DD_TRACE_SPANS_LIMIT'   => '1',
+        ]);
+    }
+
+    /**
+     * Testing the successful response code (200)
+     * root span + custom span generated in index.php script.
+     */
+    public function testResponseStatusCodeSuccess() {
+        $traces = $this->tracesFromWebRequest(
+            function () {
+                $this->call(GetSpec::create('Root', '/index.php'));
+            }
+        );
+
+        $this->assertExpectedSpans(
+            $traces, [
+                SpanAssertion::build('web.request', 'web.request', 'web', 'GET /index.php')->withExactTags([
+                    'http.method' => 'GET',
+                    'http.url' => '/index.php',
+                    'http.status_code' => '200',
+                    'integration.name' => 'web',
+                ]),
+            ]
+        );
+    }
+
+    /**
+     * Testing the successful response code (500)
+     * root span + custom span generated in index.php script.
+     */
+    public function testResponseStatusCodeError() {
```

Comment:
This test will fail as it is, as it will always return 200 instead of 500, but the good news is that it is super easy to fix it 😃.

The reason is that you cannot have 2 entry points `index.php` and `error.php` in the same test. The only entry point is defined in `getAppIndexScript()` at the begin of the class.

Here is what I suggest to do:

1. remove `error.php`
2. change `index.php` to be something similar to
```
<?php

switch ($_SERVER['REQUEST_URI']) {
    case "/success":
        http_response_code(200);
        break;
    case "/error":
        http_response_code(500);
        break;
}
``` 
3. In url use `/success` and `/error` instead of `/index.php` and `/error.php`.
4. Please note that you also have to update a few resource names/tags (e.g. `/error.php ---> /error`)

Reply:
done, thanks